src/main/java/Storage.java

```diff
@@ -15,9 +15,12 @@
  * run the program again.
  */
 public class Storage {
+    private static final String DESTINATION = "./duke.txt";
     private final Path path;
-    private static String DESTINATION = "./duke.txt";
```

Comment:
Nice work. Constant names must be all uppercase using underscore to separate words.